### PR TITLE
AC Automated Test - LRAC-8271 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LRAC-8271
https://issues.liferay.com/browse/LRAC-8932 (refactor)

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSites.macro [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+		key_percentValue = "${percentValue}";
+
+		Click(
+			key_cardName = "${cardName}",
+			locator1 = "ACSites#ACTIVITIES_CARD_GENERAL");
+
+		AssertElementPresent(
+			key_cardName = "${cardName}",
+			key_totalValue = "${totalValue}",
+			locator1 = "ACSites#SITE_ACTIVITY_VALUE_CARD");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSites.macro:15]: Hi @AfonsoHSantos, I think `viewVisitorsBehaviorMetric` macro should do the same function as that part of yours

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSites.macro [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+
+		AssertElementPresent(
+			key_cardName = "${cardName}",
+			key_percentValue = "${percentValue}",
+			locator1 = "ACSites#SITE_ACTIVITY_PERCENTAGE_CARD");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSites.macro:20]: Hi @AfonsoHSantos, see https://github.com/liferay-qa/liferay-portal/pull/757/commits/7668020b554861ffca7217a5f913debf735636a2 and talk to @LucasAcacio to define which macro to put to validate the percentages. He is also doing a test similar to yours

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase [hunk 3]
```diff
@@ -587,12 +587,38 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8271 | Test Summary: Site Activity Card: Total site visitors and % change is shown"
-	@ignore = "true"
 	@priority = "5"
 	test ShownTotalSiteVisitorsAndPercentageChangeShown {
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase:591]: Hi @AfonsoHSantos, I think this test should have data in this field in the past so when you visit the site, the **percentage** would **change**. I believe that validating that it is `0%` would not be the best way

cc: @alexcandidos

**Pull request closed without merging**